pkg/filter/utils_test.go

```diff
@@ -42,6 +42,7 @@ func TestIsSchema(t *testing.T) {
 		{tifilter.InspectionSchemaName, true},
 		{tifilter.PerformanceSchemaName, true},
 		{tifilter.MetricSchemaName, true},
+		{"tidb_cdc", true},
```

Comment:
maybe use `TiCDCSystemSchema` instead of `"tidb_cdc"` for this one

Reply:
Good catch, thank you.